File: 2185-counting-words-with-a-given-prefix/2185-counting-words-with-a-given-prefix.py
class Solution:
    def prefixCount(self, words: List[str], pref: str) -> int:
        ans  = 0
        for i in range(len(words)):
            if len(words[i])>=len(pref):
                if(pref == words[i][0:len(pref)]):
                    ans+=1
        return ans

File: 2185-counting-words-with-a-given-prefix/2185-counting-words-with-a-given-prefix.java
class Solution {
    public int prefixCount(String[] words, String pref) {
        int ans = 0;
        for(int i=0;i<words.length;i++){
            if(words[i].length()>=pref.length()){
                String sub = words[i].substring(0,pref.length());
                    if(pref.equals(sub)){
                        ans++;
                }
               
            }
        }
        return ans;
    }
}